src/pybind/mgr/telemetry/module.py

```diff
@@ -900,19 +900,11 @@ def compile_report(self, channels: Optional[List[str]] = None) -> Dict[str, Any]
             report['crashes'] = self.gather_crashinfo()
 
         if 'perf' in channels:
-            report['perf_counters_aggregated'] = self.gather_perf_counters('aggregated')
-            report['perf_counters_separated'] = self.gather_perf_counters('separated')
-
-            report['stats_per_pool'] = self.get('pg_dump')['pool_stats']
```

Comment:
can stats_per_pool be derived from stats_per_pg below? if not, we need to decide what metrics to keep here

Reply:
Thanks for pointing this out, I was too rash in removing `stats_per_pool` altogether.

I've been working on compiling a spreadsheet of all the perf channel metrics. If you check [this spreadsheet](https://docs.google.com/spreadsheets/d/1VBkQD_Jknxqx7R_8n0-XZqZ13QgP3PBQlYH1XBOiRh0/edit?usp=sharing), you can see a comparison of metrics between `stats_per_pg` and `stats_per_pool`.

In summary, we can derive these `stats_per_pool` metrics from `stats_per_pg`:
- `log_size`
- `num_pg`
- `ondisk_log_size`
- `poolid`
- `stat_sum`
We cannot derive these metrics from `stats_per_pg`, as they are specific to `stats_per_pool`:
- `acting`
- `num_store_stats`
- `store_stats`
- `up`

We could remove the redundant metrics from `stats_per_pool`, but it also might make sense to keep them there to make it easier on someone inspecting the report.

Reply:
Added the `stats_per_pool` metrics back in for now.